dd-sdk-android/src/main/json/log/log-schema.json

```diff
@@ -0,0 +1,174 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "log-schema.json",
+  "title": "LogEvent",
+  "type": "object",
+  "description": "Structure holding information about a Log",
+  "properties": {
+    "status": {
+      "type": "string",
+      "description": "The severity of this log",
+      "enum": [
+        "critical",
+        "error",
+        "warn",
+        "info",
+        "debug",
+        "trace",
+        "emergency"
+      ],
+      "readOnly": true
+    },
+    "service": {
+      "type": "string",
+      "description": "The service name",
+      "readOnly": false
+    },
+    "message": {
+      "type": "string",
+      "description": "The log message",
+      "readOnly": false
+    },
+    "logger": {
+      "type": "object",
+      "description": "Information about the logger that produced this log.",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of the logger",
+          "readOnly": false
+        },
+        "thread_name": {
+          "type": "string",
+          "description": "The thread name on which the log event was created",
+          "readOnly": true
+        },
+        "version": {
+          "type": "string",
+          "description": "The SDK version name",
+          "readOnly": true
+        }
+      },
+      "required": [
+        "name",
+        "thread_name",
```

Comment:
thread_name should not be required (e.g. if the log is a crash that comes from the NDK)